FRENCH/src/ch03-03-how-functions-work.md

```diff
@@ -312,23 +312,23 @@ $ cargo run
    Compiling functions v0.1.0 (file:///projects/functions)
     Finished dev [unoptimized + debuginfo] target(s) in 0.31 secs
      Running `target/debug/functions`
-La valeur de x est : 5
-La valeur de y est : 6
+La valeur de x est : 5
+La valeur de y est : 6
 ```
 
 <!--
 Because we called the function with `5` as the value for  `x` and `6` is passed
 as the value for `y`, the two strings are printed with these values.
 -->
 
-Comme nous avons appellé la fonction avec la valeur `5` pour `x` et `6` pour
+Comme nous avons appelé la fonction avec la valeur `5` pour `x` et `6` pour
 `y`, deux lignes sont affichées avec ces valeurs.
 
 <!--
 ### Function Bodies Contain Statements and Expressions
 -->
 
-### Corps de fonction avec des déclarations et des expressions
+### Corps de fonction avec des instructions et des expressions
```

Comment:
yes, `instructions` is a much better translation than `déclarations` !
I remember to have done this change on next chapters but i don't remember where exactly ...
thank you !